frontend/src/container/PlannedDowntime/PlannedDowntimeList.tsx

```diff
@@ -0,0 +1,340 @@
+	)}`;
+
+	return (
+		<>
+			<Collapse accordion className="collapse-list">
+				<Panel
+					header={
+						<HeaderComponent
+							duration={
+								schedule?.recurrence?.duration
+									? (schedule?.recurrence?.duration as string)
+									: getDuration(schedule?.startTime, schedule?.endTime)
+							}
+							name={defaultTo(name, '')}
+							handleEdit={(): void => {
+								setInitialValues({ ...props });
+								setModalOpen(true);
+								setEditMode(true);
+							}}
+							handleDelete={(): void => {
+								handleDeleteDowntime(id, name || '');
+							}}
+						/>
+					}
+					key={id}
+				>
+					<CollapseListContent
+						created_at={defaultTo(createdAt, '')}
+						// created_by_email="valdez@signoz.com"
+						created_by_name={defaultTo(createdBy, '')}
+						timeframe={[schedule?.startTime, schedule?.endTime]}
+						repeats={schedule?.recurrence}
+						updated_at={defaultTo(updatedAt, '')}
+						updated_by_name={defaultTo(updatedBy, '')}
+						alertOptions={alertOptions}
+					/>
+				</Panel>
+			</Collapse>
+			<div className="view-created-at">
+				<CalendarClock size={14} />
+				<Typography.Text>{formattedDateAndTime}</Typography.Text>
+			</div>
+		</>
+	);
+}
+
+export type DowntimeSchedulesTableData = DowntimeSchedules & {
+	alertOptions: DefaultOptionType[];
+};
+
+export function PlannedDowntimeList({
+	downtimeSchedules,
+	alertOptions,
+	setInitialValues,
+	setModalOpen,
+	handleDeleteDowntime,
+	setEditMode,
+	searchValue,
+}: {
+	downtimeSchedules: UseQueryResult<
+		AxiosResponse<PayloadProps, any>,
+		AxiosError<unknown, any>
+	>;
+	alertOptions: DefaultOptionType[];
+	setInitialValues: React.Dispatch<
+		React.SetStateAction<Partial<DowntimeSchedules>>
+	>;
+	setModalOpen: React.Dispatch<React.SetStateAction<boolean>>;
+	handleDeleteDowntime: (id: number, name: string) => void;
+	setEditMode: React.Dispatch<React.SetStateAction<boolean>>;
+	searchValue: string | number;
+}): JSX.Element {
+	const columns: TableProps<DowntimeSchedulesTableData>['columns'] = [
+		{
+			title: 'Downtime',
+			key: 'downtime',
+			render: (data: DowntimeSchedulesTableData): JSX.Element =>
+				CustomCollapseList({
+					...data,
+					setInitialValues,
+					setModalOpen,
+					handleDeleteDowntime,
+					setEditMode,
+				}),
+		},
+	];
+
+	const tableData = (downtimeSchedules.data?.data?.data || [])
+		?.filter(
+			(data) =>
+				data?.name?.includes(searchValue.toLocaleString()) ||
+				data?.id.toLocaleString() === searchValue.toLocaleString(),
+		)
+		.map?.((data) => {
+			const specificAlertOptions = getAlertOptionsFromIds(
+				data.alertIds || [],
+				alertOptions,
+			);
+
+			return { ...data, alertOptions: specificAlertOptions };
+		});
+
+	return (
+		<Table<DowntimeSchedulesTableData>
+			columns={columns}
+			className="planned-downtime-table"
+			bordered={false}
+			dataSource={tableData || []}
+			loading={downtimeSchedules.isLoading || downtimeSchedules.isFetching}
```

Comment:
use ghost loading state here

Reply:
Sure, will add this in followup PR.
Thanks!